src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -155,7 +155,7 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
                         (token, notifications) => this.DiscoverProjects(controller, notifications)),
 
                 new ProgressStepDefinition(Strings.BindingProjectsDisplayMessage, StepAttributes.BackgroundThread,
-                        (token, notifications) => this.DownloadQualityProfileAsync(controller, notifications, this.GetBindingLanguages(), token)),
+                        (token, notifications) => this.DownloadQualityProfileAsync(controller, notifications, this.GetBindingLanguages(), token).GetAwaiter().GetResult()),
```

Comment:
Are there any other _void_ methods called in _ProgressStepDefinition_ steps?

Reply:
Doesn't seem so.